### ISSUE TITLE
Big Questions: critique by Willie Wong on Google+

### ISSUE THREAD

**Issue by theronhitchman** [theronhitchman/differential-geometry#1]
I would suggest re-writing 2 and 3. The point of view differentiating intrinsic and extrinsic geometry is not something that can be appreciated by someone who haven't already studied some differential geometry. Basically, to understand 3 (as you wrote it) you have to have some appreciation of manifolds defined absent an external embedding.

So 2. I think is largely okay, though I would write it as "what can we learn about the geometry of an object by looking at how it sits in space" and avoid the whole business of outside/inside, which may call up incorrect preconceptions.

Point 3. however I would rephrase with the big question being more like "Can we distinguish geometric objects without referring to how they sit in space?" The second explanatory part you wrote works equally well with this.

Another thing that may help is to build some of this into point 1. The extrinsic point of view is that of restricting calculus to "suitably nice subsets" and that it will work. The intrinsic point of view is that of building calculus up by looking at locally Euclidean spaces. In fact, for point 1 I would also drop the parenthetical reference to smooth: since you are asking about how we can study the shape of objects using calculus, smoothness should be part of the answer, and not part of the question.

Points 4 and 5 could perhaps be merged... though I wouldn't know until I see the topical learning objectives.

I am also a bit confused by your local-versus-global theme. Can you give me of an example of a global property you are thinking about? (Incidentally there's also a grammatical error in the statement of that theme.)

Since you are only dealing with curves and surfaces, maybe you can also drop the "curves versus surfaces" theme and replace all references to "geometric objects" by references to "curves and surfaces". Or say something up front before the first question like:

"For each of the two types of geometric objects studied in this course---curves and surfaces---we ask the following Big Questions:"

Basically my point is that there's not need to unify the two classes into one (geometric objects) when at the end of the day, the theme is that the two classes often behave differently (so why merge them to start with?)﻿

**Comment by theronhitchman** [comment]
[conversation thread here](https://plus.google.com/u/0/+TheronHitchman/posts/8RDDb578PPy)
